r/data-raw/codegen.R

```diff
@@ -194,10 +194,16 @@ arrow_exports_cpp <- glue::glue('
 
 {feature_available("arrow")}
 
+{feature_available("dataset")}
```

Comment:
It might be good to make these bits iterate over `features` rather than copy-paste, just to help with future features.

Reply:
Done in c8d390d